src/coreclr/jit/morph.cpp

```diff
@@ -12076,7 +12079,7 @@ GenTree* Compiler::fgMorphFieldAssignToSimdSetElement(GenTree* tree)
     unsigned    simdSize        = 0;
     GenTree*    simdStructNode  = getSIMDStructFromField(tree->gtGetOp1(), &simdBaseJitType, &index, &simdSize);
 
-    if (simdStructNode != nullptr)
+    if ((simdStructNode != nullptr) && IsBaselineSimdIsaSupported())
```

Comment:
Is this check for `IsBaselineSimdIsaSupported` still needed?

Reply:
no, I thought I removed it, will fix.

Reply:
Removed, I had missed the one from SetElement but gotten the one in GetElement